sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/administration/PersonalizerAdministrationAsyncClient.java

```diff
@@ -512,23 +512,8 @@ public Mono<Response<BinaryData>> exportModelWithResponse(boolean isSigned) {
      * @return the model file generated by Personalizer service.
      */
     Mono<Response<BinaryData>> exportModelWithResponse(boolean isSigned, Context context) {
-        return service.getModels().getWithResponseAsync(isSigned, context).flatMap(streamResponse -> {
-            return BinaryData.fromFlux(streamResponse.getValue())
-                .flatMap(binaryData -> {
-                    Response<BinaryData> response = new SimpleResponse<>(
-                        streamResponse.getRequest(),
-                        streamResponse.getStatusCode(),
-                        streamResponse.getHeaders(),
-                        binaryData);
-                    return Mono.just(response);
-                }).doFinally(ignored -> {
-                    try {
-                        streamResponse.close();
-                    } catch (Exception e) {
-                        logger.logThrowableAsError(e);
-                    }
-                });
```

Comment:
Do we know why this was done previously and why it's no longer needed?

Reply:
REST layer now generated with BinaryData instead of StreamResponse